plsync/planetlab/model.py

```diff
@@ -30,6 +30,17 @@ def pl_interface(host_index, v4prefix):
     interface['ip']         = '%s.%d' % (net_prefix, mlab_offset)
     return interface
 
+def split_prefix(v4prefix):
```

Comment:
I would like to refactor almost all of this file. Your suggestion looks nice. Ideally, I'd like to get away from needing to call this method any more than once per network. Since you already added a "LGTM", it sounds like you're ok with leaving this to a future revision.